app/helpers/application_helper.rb

```diff
@@ -1,2 +1,15 @@
 module ApplicationHelper
+
+  def districts_options
```

Comment:
you may want to use [select_tag](http://apidock.com/rails/ActionView/Helpers/FormTagHelper/select_tag) instead, as mentioned in jacqui's pr..https://github.com/wdi-hk-9/Project-SushiMeAway/pull/11#issuecomment-153039919